src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE hibernate-configuration PUBLIC
```

Comment:
it looks like we don't need this file...

Reply:
"Your task is to configure Hibernate in this project and..." I understood this like I need to create cfg file...

Reply:
we don't actually need this file but it's ok